Formula/crowdin.rb

```diff
@@ -14,6 +14,13 @@ def install
   end
 
   test do
-    system bin/"crowdin"
+    generate_output = shell_output("#{bin}/crowdin generate").chomp
+    ohai generate_output
```

Comment:
I don't think we should use `ohai` (except maybe for debugging)

Reply:
100% agree. It was there for testing & debugging, and I forgot to remove it. I just force pushed to remove that line.